Obedient-Lobsters/BangazonAPI
# Creation of Customer Model

# Creation of Customer Model
# Description
 Created Customer Model for # 1, will still need to create the controller for the issue. Property LastLogin is not tracked or modified in other issues of tickets. It is okay to leave the value that reflects account creation. This model is dependent on Product and PaymentType models and is needed for Product, Order and CustomerPayment.
 Fixes  (#1 )
 ## Type of change
 Please delete options that are not relevant.
 # Testing Instructions
-Ensure that Product.cs and PaymentType.cs is included in the Model folder

 -In the directory BangazonAPI, use command "dotnet run" to run program. The server should then start and run without errors.

-Make sure all names line up with ERD.(https://drive.google.com/file/d/1Qo9O2oUezNd13Wsr0_VUbUtwwDo4-oAt/view)

 # Checklist:
